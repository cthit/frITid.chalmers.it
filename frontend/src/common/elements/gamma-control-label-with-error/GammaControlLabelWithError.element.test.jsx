import React from "react";
import { shallow } from "enzyme";
import GammaControlLabelWithError from "./";

describe("<GammaControlLabelWithError/>", () => {
  test("Shallow render of <GammaControlLabelWithError/> with error", () => {
    const wrapper = shallow(
      <GammaControlLabelWithError
        label="This is a gamma control label with error"
        error={true}
      />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
