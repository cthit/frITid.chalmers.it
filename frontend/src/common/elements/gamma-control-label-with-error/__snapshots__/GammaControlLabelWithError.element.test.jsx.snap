// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GammaControlLabelWithError/> Shallow render of <GammaControlLabelWithError/> with error 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Styled(Component)
    error={true}
    label="This is a gamma control label with error"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "className": "sc-bdVaJa kOJsyC",
      "error": true,
      "label": "This is a gamma control label with error",
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "className": "sc-bdVaJa kOJsyC",
        "error": true,
        "label": "This is a gamma control label with error",
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
